### DISCUSSION TITLE
Folder rename

### DISCUSSION THREAD

**Discussion by CoolGoose** [Lukasss93/laravel-larex#40] [General]
Hello there... :) 

Is there a reason why the laravel exporter does a `remapping` of the folder name and it doesn't allow `-` as a separator ?

**Comment by Lukasss93** [comment]
Because of this:
https://laravel.com/docs/11.x/localization#defining-translation-strings

![immagine](https://github.com/Lukasss93/laravel-larex/assets/4071613/d19bad55-416b-4dbb-859b-9526934054fb)

However, we can find a way to make it more configurable 🧐

**Comment by CoolGoose** [comment]
Hey @Lukasss93 I did a small PR, happy to take it in whatever direction you think it's best (config file, extend it to the other exporters etc)
